devices/XPT2046/Point.cs

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) .NET Foundation and Contributors
+// Portions Copyright (c) Microsoft Corporation.  All rights reserved.
+// See LICENSE file in the project root for full license information.
+
+namespace Iot.Device.Xpt2046
+{
+    /// <summary>
+    /// A touch point.
+    /// </summary>
+    public struct Point
```

Comment:
sorry for seen this but we do have the nanoFramework.Graphics.Core nuget which implements the points: https://github.com/nanoframework/nanoFramework.Graphics/blob/main/nanoFramework.Graphics.Core/System/Drawing/Point.cs

Please use that one instead of your own struct. That will allow to use it properly with the screens and have more flexibility. So, it's about adding this nuget, removing this file and adding the import in your code. It should just work transparently! As a bonus, you get all the point operations and more!